prime-angular-frontend/src/app/modules/adjudication/pages/site-overview/site-overview.component.ts

```diff
@@ -185,11 +155,11 @@ export class SiteOverviewComponent extends SiteRegistrationContainerComponent im
     });
   }
 
-  private initForm({ pec }: Site) {
+  private initForm({ pec }: Site): void {
     this.form.patchValue({ pec });
   }
 
   private checkPecIsAssignable(): (value: string) => Observable<boolean> {
-    return (value: string) => this.siteResource.pecAssignable(this.site.id, value);
+    return (value: string) => this.siteResource.pecAssignable(this.route.snapshot.params.sid, value);
   }
 }
```

Comment:
Health Authority Site Overview ✔️ 
Health Authority Site Events ✔️ 
Health Authority Site Documents ✔️ ⚠️ I can upload the documents, but I can't see the documents anywhere except in the upload document page, is that expected behavior?
Health Authority Site Notes ✔️ 
Health Authority Site Actions ✔️ ⚠️ I can see the actions, however I cannot approve the site. I am not sure if it is because we don't have the field to assign Site ID such as the one we have for community sites where we have to provide a Site ID before we can approve it. If make the site editable and login as the authorized user and edit the site ID, then going back as an admin I can approve the site.
Site Overview ✔️ 
Site Actions ✔️

Reply:
Health Authority Site Documents ✔️ ⚠️ I can upload the documents, but I can't see the documents anywhere except in the upload document page, is that expected behavior?

Yes

Actions: I have moved the checks so only community sites approvals are blocked missing a Pec. There was no mention of being able to assign pec/siteid for admin for ha sites so that might be flushed out later.